### PR TITLE
[ADD] client to calendar event form and required

### PR DESCRIPTION
**Pull request by eilst**

### CODE REVIEW

File: project_resource_calendar/models/calendar_event.py [hunk 1]
```diff
@@ -179,3 +185,13 @@ def print_calendar_report(self):
         return self.env.ref(
             'project_resource_calendar.calendar_event_report'
         ).report_action(self)
+
+    @api.model
+    def create(self, vals):
```
Comment (rimbendhaou) [inline project_resource_calendar/models/calendar_event.py:199]: @eilst what if we modify (we use write method) and we delete the client from the participants list? we will not do this verification(verify_client_in_participants)?

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_resource_calendar/models/calendar_event.py (hunk 1 above)
Comment (eilst) [reply project_resource_calendar/models/calendar_event.py:199]: Done, right, the validation should also be done when writing

File: project_event/models/calendar_event.py [hunk 2]
```diff
@@ -21,3 +21,17 @@ class CalendarEvent(models.Model):
...
+        store=True,
+    )
+
+    @api.one
+    @api.depends('client_id')
+    def _get_client_tag_id(self):
+        if self.client_id:
+            self.client_tag = self.client_id.tag_id
+        else:
+            self.client_tag = None
```
Comment (rimbendhaou) [inline project_event/models/calendar_event.py]: PEP8 new line after None

File: project_event/models/calendar_event.py [hunk 3]
```diff
@@ -21,3 +21,17 @@ class CalendarEvent(models.Model):
         string='Client Type',
     )
     color = fields.Char(related='category_id.color')
+    client_tag = fields.Many2one(
```
Comment (rimbendhaou) [inline project_event/models/calendar_event.py:24]: If it is a readonly field it is better to put it as a related field

Comment (eilst) [reply project_event/models/calendar_event.py:24]: Right, done

File: project_event/models/calendar_event.py (hunk 2 above)
Comment (eilst) [reply project_event/models/calendar_event.py]: Done

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**